test/system/tools/fds/Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Testing directory should not be added to the repo. Maybe we need a rule in the .gitignore of FDS

Reply:
👍
 `>git rm test\system\tools\fds\Testing -r`